packages/min-100-layout/style.module.css

```diff
@@ -0,0 +1,10 @@
+.root {
```

Comment:
These styles are based on a [similar solution implemented for Learn](https://github.com/hashicorp/learn/blob/2ded2a4d21e4649c9bdcf6afc0824fd23cfcbba4/pages/style.css#L266).